### DISCUSSION TITLE
Ошибка после последнего обновления

### DISCUSSION THREAD

**Discussion by mihail-cornev** [totumonline/totum-ru-issues-and-discussions#579] [ОШБК!] [answered]
![Screenshot_19](https://github.com/user-attachments/assets/ea7e49c3-1887-4b80-a197-c85ebc829aa1)

По идее последняя часть кода даёт ошибку, но я не пойму почему.

Код действия по нажатию кнопки:
a1= : while(action: $check; action: $insert; action: $set1; action: $set2; action: $open)

// Проверяем если поля заполнены
	check: listReplace(list: $act_list; action: $action; value: "val")
	act_list: rowListCreate(field: "field" = $field_list; field: "name" = $field_list_title)
		field_list: listCreate(item: #h_name_jur_full; item: #h_form)
		field_list_title: listCreate(item: "Официальное название"; item: "Форма предприятия")

// Выдаёт ошибку, если поле не заполнено		
	action: if(condition: $#val[field] = ""; then: $err)
		err: errorException(text: str`"Не заполнено поле:" ++ $#val[name]`)
	
// Если все поля заполнены, добавляет строку в цикл таблицы Контрагенты
	insert: insert(table: 'contragents_'; inserts: "company")
	set1: setList(table: 'contragents_tab_info'; cycle: $#company[0]; field: 'h_type' = 2; field: 'h_name_jur_full' = #h_name_jur_full; field: 'h_name_jur' = #h_name_jur; field: 'h_form' = #h_form; field: 'h_activity' = #h_activity; log: true)
	set2: setList(table: 'contragents_tab_details'; cycle: $#company[0]; field: 'h_idn' = #h_idno; log: true)
	**open: linkToButtons(title: "Открыть данные контрагента?"; buttons: $b10; refresh: true)
			b10: rowCreate(field: "text" = "Открыть"; field: "code" = $code1; field: "icon" = "link"; field: "background" = "#FFE486")
			code1: linkToTable(table: 'contragents_tab_details'; cycle: $#company[0]; target: "iframe"; width: "85vw"; refresh: true)**

**Comment by mihail-cornev** [comment]
![image](https://github.com/user-attachments/assets/2ce91cb3-d38d-440c-8990-0a2f3c078337)

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Мне каж buttons должны быть rowList по сути названия параметра. А там row

**Reply by mihail-cornev** [reply]
спасибо, исправил - работает
Точно нужен ИИ чтобы проверять код )) выучить всю документацию и помнить нюансы непросто
